addon/components/frost-button.js

```diff
@@ -227,12 +246,17 @@ export default Component.extend(PropTypeMixin, {
     }
   }),
 
+  // FIXME: jsdoc
```

Comment:
While in principle I agree with you, in practice I don't. I've found it's much more difficult to enforce "put JSDoc comments on appropriate functions" vs. "put JSDoc comments on all functions" If the function is simple, the doc can be one-line, if it has parameters, they should be documented though. 

It's also very easy for a function that was simple, to suddenly become less so. By having the jsdoc comment there, we have lint rules that ensure it's accurate (all params are covered, return type is covered, etc). If it doesn't have anything, the function can creep in complexity w/o being documented.